src/main/java/org/entur/gbfs/HttpUtils.java

```diff
@@ -36,15 +36,16 @@ public static InputStream getData(String uri, Map<String, String> headers) throw
     }
 
     public static InputStream getData(
-        URI uri, long timeout, Map<String, String> requestHeaderValues
+            URI uri, Long timeoutConnection, Map<String, String> requestHeaderValues
```

Comment:
This value is used both for connection timeout and socket timeout. This name change is therefore inaccurate. Should it be possible to override both maybe? Otherwise, keep the original name.